openmicroscopy/openmicroscopy
# Right click

Add missing option to menu in centre panel
To test:
 * Browse a dataset
 * Select an image in the centre panel
 * Check the Download option
 * Dialog should pop-up
 * Select where to download the file.
 * Click download, Check that the file is downloaded

see https://trac.openmicroscopy.org/ome/ticket/9779

--no-rebase
